harvard-stellenbosch-university.csl

```diff
@@ -60,10 +60,6 @@
         </choose>
       </substitute>
     </names>
-  </macro>
-  <macro name="author-short">
-    <names variable="author">
-      <name form="short" and="symbol" delimiter-precedes-last="never" et-al-min="" et-al-use-first="" et-al-subsequent-min="3" et-al-subsequent-use-first="1" initialize-with="." sort-separator=" "/>
```

Comment:
My apologies if this is getting a bit tedious, but you deleted four lines, which resulted in a style that is still invalid CSL.

Can you accept this pull request, which should fix this?: https://github.com/Yusufras/styles/pull/1